### ISSUE TITLE
Discarding outermost pixels

### ISSUE THREAD

**Issue by vineetbansal** [ComputationalCryoEM/aspyre#14]
The paper suggests `..we discard the 100 outermost pixels.`, but the following snippet discards 99 pixels from the top and left, and 100 pixels from the bottom and right, reducing a 4096x4096 image to 3897x3897 in size.

https://github.com/ComputationalCryoEM/aspyre/blob/c4f34a170d1dcc681356cbac319ee09f3774cfa6/src/aspyre/apple/picking.py#L53

The margins will be moved to the config.json file soon, and if/when this is corrected, the unit tests would have to be rewritten to pick the updated centers. At that point this can be closed.

**Comment by junchaoxia** [comment]
@vineetbansal  Yes. I believe this is a typo of indexing.  It should not have large effect to the particle picking except the centers could be shifted.

**Comment by vineetbansal** [comment]
What's interesting is that after the centers are found, they are offset by 99 in the snippet below (instead of 100), so the original author realizes his/her error without fixing it (or perhaps they did intend to use 99 for whatever reason):

https://github.com/ComputationalCryoEM/aspyre/blob/c4f34a170d1dcc681356cbac319ee09f3774cfa6/src/aspyre/apple/picking.py#L288
